examples/transforms_demo_2d.ipynb

```diff
@@ -0,0 +1,271 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2D image transformation demo\n",
+    "\n",
+    "This demo shows how to apply 2D transforms in MONAI.\n",
+    "Main features:\n",
+    "    - Random elastic transforms implemented in native Pytorch\n",
+    "    - Easy-to-use interfaces that are designed and implmented in the pythonic way\n",
+    "    \n",
+    "Find out more in MONAI's wiki page: https://github.com/Project-MONAI/MONAI/wiki"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Before running this demo\n",
+    "**please download the GLAS (gland segmentation in histology images) challenge data from\n",
```

Comment:
Suggest to add ":" and "\n" at end of this line, then the notebook style will be normal.